yb-voyager/cmd/analyzeSchema.go

```diff
@@ -918,6 +918,9 @@ sqlParsingLoop:
 			} else if matches := dollarQuoteRegex.FindStringSubmatch(currLine); matches != nil {
 				dollarQuoteFlag = 1 //denotes start of the code/body part
 				codeBlockDelimiter = matches[0]
+			} else if strings.Contains(currLine, "BEGIN ATOMIC") {
```

Comment:
This is also fixed in https://github.com/yugabyte/yb-voyager/pull/2201/files ? 
Is that a prerequisite for this PR?

Reply:
yes other wise analyze-schema/assessment won't be able to report the case as our parser will not work